syz-manager/manager.go

```diff
@@ -672,11 +672,10 @@ func (mgr *Manager) saveCrash(crash *manager.Crash) bool {
 		resp, err := mgr.dash.ReportCrash(dc)
 		if err != nil {
 			log.Logf(0, "failed to report crash to dashboard: %v", err)
-		} else {
-			// Don't store the crash locally, if we've successfully
-			// uploaded it to the dashboard. These will just eat disk space.
-			return mgr.cfg.Reproduce && resp.NeedRepro
 		}
+		// Don't store the crash locally even if we failed to upload it.
+		// There is 0 chance that one will ever look in the crashes/ folder of those instances.
+		return mgr.cfg.Reproduce && resp.NeedRepro
```

Comment:
That helped to have at least some visibility into such errors. If this happens more often than episodically, it's a real problem (we lost bugs/repros).
How can we keep some visibility into these errors? Say, if the rate suddenly increases?
Is using `log.Errorf` above a good idea? Or will that produce too many alerts?

Reply:
Currently we can also estimate the rate retroactively. E.g. check now how may was lost in the past year.

Reply:
If we want to estimate the rate, it's more reliable to look at the Cloud logs. I've searched `failed to report crash to dashboard` across all syzbot instances and it showed 35 times over the last 7 days (i.e. 5 times a day on average).

If we turn them to `log.Errorf`, that'd be quite a few. Most of them are `operating on too many entity groups in a single transaction`, so maybe we could address that and the rate will be okay...

Reply:
We never really grep these logs for messages, but we should, filed #5428.
Grepping logs is also fragile, tomorrow somebody decides to improve the error message and all alerting falls apart. Error messages are changed all time, just for yesterday: #5422, #5423.